<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSSH</class>
 <widget class="QWidget" name="EditSSH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">EditSSH</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="Private_Key">
     <property name="text">
      <string>Private Key</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="host_key"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="private_key_path"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="private_key_passphrase"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="user"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="host_key_algorithms"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="private_key"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="Private_Key_Path">
     <property name="text">
      <string>Private Key Path</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="Private_Key_Passphrase">
     <property name="text">
      <string>Private Key Passphrase</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="Password">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="User">
     <property name="text">
      <string>User</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="Host_Key">
     <property name="text">
      <string>Host Key</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="password"/>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="client_version"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="Host_Key_Algorithms">
     <property name="text">
      <string>Host Key Algorithms</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="Client_Version">
     <property name="text">
      <string>Client Version</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>user</tabstop>
  <tabstop>password</tabstop>
  <tabstop>private_key</tabstop>
  <tabstop>private_key_path</tabstop>
  <tabstop>private_key_passphrase</tabstop>
  <tabstop>host_key</tabstop>
  <tabstop>host_key_algorithms</tabstop>
  <tabstop>client_version</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
